### PR TITLE
Update on contributing documentation

### PR DESCRIPTION
**Pull request by emmanueletukudo**
### Scope & Purpose

*(Please describe the changes in this PR for reviewers, motivation, rationale - **mandatory**)*

This PR adds documentation on how to fix Cross-Origin-Policy errors when running ArangoDB on a development server.
 
- [ ] :hankey: Bugfix
- [ ] :pizza: New feature
- [ ] :fire: Performance improvement
- [ ] :hammer: Refactoring/simplification

### Checklist

- [ ] Tests
  - [ ] **Regression tests**
  - [ ] C++ **Unit tests**
  - [ ] **integration tests**
  - [ ] **resilience tests**
- [ ] :book: CHANGELOG entry made
- [ ] :books: documentation written (release notes, API changes, ...)
- [ ] Backports
  - [ ] Backport for 3.9: *(Please link PR)*
  - [ ] Backport for 3.8: *(Please link PR)*
  - [ ] Backport for 3.7: *(Please link PR)*

#### Related Information

*(Please reference tickets / specification / other PRs etc)*

- [x] Docs PR: 
- [ ] Enterprise PR:
- [ ] GitHub issue / Jira ticket:
- [ ] Design document:

### CODE REVIEW

**Comment by jsteemann** [comment]
@emmanueletukudo : I guess this PR is there to update the contributing notes, but it also changes other files. Is that intentional?

**Comment by emmanueletukudo** [comment]
@jsteemann it wasn't intentional, I guess I mistakenly move changes from my previous branch to the intended branch.

**Comment by emmanueletukudo** [comment]
> @jsteemann it wasn't intentional, I guess I mistakenly move changes from my previous branch to the intended branch.

I'll close this PR and create a new one.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -621,28 +649,28 @@ The symbolserver over at https://download.arangodb.com/symsrv_arangodbXX/ is
...
-    - arangobench
-    - arangodump
-    - arangoexport
-    - arangoimport
-    - arangorestore
-    - arangosh
-    - arangovpack
+- arango - the basic arangodb library needed by all components
+- arango_v8 - the basic V8 wrappers needed by all components
+- arangod - the server process
```
Comment (dothebart) [inline CONTRIBUTING.md:654]: while you're editing, this is probably an old remain.
please change 
arango -> lib
arango_v8 lib/v8

**Pull request closed without merging**